src/components/transactionsV2/transactionsOverviewHeader/transactionsOverviewHeader.js

```diff
@@ -70,76 +71,87 @@ class transactionsHeader extends React.Component {
+              delegate={delegate}
+              account={this.props.account}
+              toggleActiveToken={this.props.toggleActiveToken}
+              />
+            <div className={`${styles.buttonsHolder}`}>
+              <Link to={`${routes.send.path}?wallet&recipient=${address}`}
+                className={'send-to-address'}>
+                  <SecondaryButtonV2>
+                    {t('Send {{token}} to this Account ', { token: activeToken })}
+                  </SecondaryButtonV2>
+              </Link>
+              <span
+                ref={this.setDropownRefs}
+                data-name={'bookmarkDropdown'}
+                className={`${styles.bookmarkContainer} bookmark-account`}>
+              { isBookmark ? (
+                <SecondaryButtonV2
+                  className={`${styles.bookmarkButton}`}
+                  onClick={
+                    /* istanbul ignore next */
+                    () => this.toggleDropdown('bookmarkDropdown')
+                  }>
+                  {t('Account bookmarked')}
                 </SecondaryButtonV2>
-            </Link>
-            <span
-              ref={this.setDropownRefs}
-              data-name={'bookmarkDropdown'}
-              className={`${styles.bookmarkContainer} bookmark-account`}>
-            { isBookmark ? (
-              <SecondaryButtonV2
-                className={`${styles.bookmarkButton}`}
-                onClick={
-                  /* istanbul ignore next */
-                  () => this.toggleDropdown('bookmarkDropdown')
-                }>
-                {t('Account bookmarked')}
-              </SecondaryButtonV2>
-            ) : (
-              <PrimaryButtonV2 onClick={() => this.toggleDropdown('bookmarkDropdown')}>
-                {t('Bookmark account')}
-              </PrimaryButtonV2>
-            )}
-            <DropdownV2
-              showDropdown={this.state.shownDropdown === 'bookmarkDropdown'}
-              className={`${styles.followDropdown}`}>
-                <Bookmark
-                  token={activeToken}
-                  delegate={delegate}
-                  address={address}
-                  detailAccount={detailAccount}
-                  isBookmark={isBookmark} />
-              </DropdownV2>
-            </span>
+              ) : (
+                <PrimaryButtonV2 onClick={() => this.toggleDropdown('bookmarkDropdown')}>
+                  {t('Bookmark account')}
+                </PrimaryButtonV2>
+              )}
+              <DropdownV2
+                showDropdown={this.state.shownDropdown === 'bookmarkDropdown'}
+                className={`${styles.followDropdown}`}>
+                  <Bookmark
+                    token={activeToken}
+                    delegate={delegate}
+                    address={address}
+                    detailAccount={detailAccount}
+                    isBookmark={isBookmark} />
+                </DropdownV2>
+              </span>
```

Comment:
It's possible to use the `OutsideClickHandler` to wrap the dropdowns, so it's not needed to handle the refs manually.